src/components/list/EntryList.js

```diff
@@ -35,6 +58,7 @@ const EntryList = forwardRef(
     { locations, onEntryClick, onEntryMouseEnter, onEntryMouseLeave, ...props },
     ref,
   ) => {
+    const { distanceUnit } = useSelector((state) => state.settings)
```

Comment:
You should be more specific with the selector.

`const distanceUnit = useSelector((state) => state.settings.distanceUnit)`